vtysh/vtysh.c

```diff
@@ -2548,7 +2548,8 @@ DEFUNSH(VTYSH_ALL, vtysh_log_facility, vtysh_log_facility_cmd,
 }
 
 DEFUNSH(VTYSH_ALL, no_vtysh_log_facility, no_vtysh_log_facility_cmd,
-	"no log facility [FACILITY]", NO_STR
+	"no log facility [<kern|user|mail|daemon|auth|syslog|lpr|news|uucp|cron|local0|local1|local2|local3|local4|local5|local6|local7>]",
```

Comment:
I doubt it would work. You should repeat `"Syslog facility"` for every facility you listed.

Reply:
Why shouldn't it work? It's exactly the same implementation as in the library. You can check lib/command.c:2579.

Reply:
You should use `LOG_FACILITY_DESC` instead.

Reply:
Thanks, now I see the problem. Didn't understand from the first comment that it's about the help string.

Reply:
Fixed.